packages/components/psammead-radio-schedule/src/testHelpers/helper.jsx

```diff
@@ -88,6 +88,8 @@ export const renderRadioSchedule = ({
     timezone={timezone}
     script={script}
     service={service}
+    nextLabel={dir === 'rtl' ? 'مباشر' : 'NEXT'}
```

Comment:
Can we move this into a constant as done above with the `renderProgramCard`?

Reply:
Sure thing